build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+echo "GRPC in $1"
+
+mkdir -p .build
+cd .build
+
+cmake -DGRPC_INSTALL_DIR=$1 ..
+cmake --build . -- -j5
```

Comment:
GitHub detects improper line endings in several files within this PR. Please fix those.

We address CRLF issues by using `core.autocrlf` Git configuration property. Or — as an alternative — use the IDEA (or CLion) setting.

Reply:
Removed the scripts, until they are ready.